job/job.go

```diff
@@ -34,6 +35,34 @@ type Job struct {
 	instance *packet.Instance
 }
 
+var tinkWorkflowClient = client.GetWorkflowsFromTink
+
+// discoverWorkflowForMac fetches the workflow for a given MAC
+// hwID is the hardware/worker ID corresponding to the MAC
+func discoverActiveWorkflowForMac(hwID string) (bool, error) {
```

Comment:
please use `MAC` instead of `Mac` in the name and comment.

Reply:
Actually nm, lets go with @gianarb's comment since it makes more sense and this func doesn't even use MAC itself anyway.

Reply:
i have renamed this method.